python/requirements.txt

```diff
@@ -9,3 +9,11 @@ grandiso
 
 # revng.api
 cffi
+
+# revng.restapi
+Flask
+Werkzeug
+ariadne
+aiodataloader
+Jinja2
+xdg
```

Comment:
Do we have a companion PR for orchestra?

Reply:
https://github.com/revng/orchestra/pull/77